blog/_posts/2018-10-03-scaladoc-tables.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: blog-detail
+post-type: blog
+by: Janek Bogucki
+title: "Introducing Scaladoc Tables"
+---
+
+Scala 2.12.7 adds support for markdown tables in Scaladoc comments.
+
+### A Simple Example
+
+The markdown for tables follows [GitHub Markdown Flavored Markdown](https://github.github.com/gfm/) with some restrictions.
+
+This table shows all three available table elements,
+
+```
+| Trait | Implementations |
+| ---   | ---             |
+| Set   | HashSet, HashTrieSet |
+| Seq   | List, Vector, Empty  |
+| Map   | HashTrieMap, ListMap |
+```
+
+A table is comprised of,
+
+ - A header row
+ - A delimiter row
+ - Zero or more data rows
+
+This is how the above markdown renders,
+
+<img alt="Example Table 1"
+     src="/resources/img/blog/scaladoc-tables/example-table-1.png">
```

Comment:
Jekyll supports an image syntax

```markdown
![Example Table 1](/resources/img/blog/scaladoc-tables/example-table-1.png)
```

Reply:
@ashawley Thanks! Changes made to both images.